src/sparseml/pytorch/image_classification/train.py

```diff
@@ -628,19 +645,20 @@ def train(
     :param rank: The rank of the process
     """
 
-    # Baseline eval run
-    trainer.run_one_epoch(
-        mode="validation",
-        max_steps=debug_steps,
-        baseline_run=True,
-    )
+    if not trainer.one_shot:
+        # Baseline eval run
+        trainer.run_one_epoch(
+            mode="validation",
+            max_steps=debug_steps,
+            baseline_run=True,
+        )
 
     if not eval_mode:
```

Comment:
let's change this to `... and not trainer.one_shot` so all the `starting training...` logging doesn't print for one shot. can move the final save step at the end of this block out with another check for `eval_mode`